packages/blockchain-link-types/src/common.ts

```diff
@@ -61,8 +61,8 @@ export interface ServerInfo {
     testnet: boolean;
     version: string;
     decimals: number;
-    blockHeight: number;
-    blockHash: string;
+    blockHeight: number | undefined;
+    blockHash: string | undefined;
```

Comment:
I would keep it defined and in case it comes undefined, I would put there e.g. genesis block and hash